avocado/test.py

```diff
@@ -548,7 +548,8 @@ def action(self):
         try:
             test_params = {str(key): str(val)
                            for key, val in self.params.iteritems()}
-            result = process.run(self.path, verbose=True, env=test_params)
+            result = process.run(pipes.quote(self.path), verbose=True,
```

Comment:
Thanks for pointing this bug, it's inconvenient to scape like this `process.run('/home/rmoura/true\ or\ false')`, let's do it, but I would like to see a test case for using space in the name of the command, so we can merge it.